Confus/ConfusServer/TeamScore.h

```diff
@@ -17,6 +20,8 @@ namespace ConfusServer
         int m_RedTeamScore = 0;
         /// <summary> The Connection class instance to send the score with. </summary>
         Networking::Connection* m_Connection;
+        /// <summary> Reset callback function </summary>
+        std::function<void()> m_Callback;
```

Comment:
You might want to make the name a bit more descriptive. Callback could be a callback for anything, even with the documentaiton provided, it's still quite unclear to me as to when this is called